P5/Source/Specs/post.xml

```diff
@@ -0,0 +1,261 @@
+                <p>
+                    <ref corresp="#A02" target="https://twitter.com/textarchiv">
+                        <w type="ADRIND">@</w><w type="ADR">textarchiv</w></ref> Erfurter
+                    Europäische Welt- und Staats-Geschichte: 20 Ausgaben von 1744 aus dem MKHZ
+                    bereits in DTAQ <ref target="http://www.deutschestextarchiv.de/dtaq/about">
+                        http://www.deutschestextarchiv.de/dtaq/about</ref>
+                    <ref target="https://twitter.com/hashtag/dtaclarin14?src=hash"><w type="hashtag"
+                            >#dtaclarin14</w></ref>
+                </p>
+            </post>
+        </egXML>
+    </exemplum>
+    <exemplum type="Chat2CLARIN" xml:lang="en" versionDate="2021-09-07">
+        <p>example from Wikipedia discussion</p>
+        <egXML xmlns="http://www.tei-c.org/ns/Examples">
+            <post type="standard" generatedBy="human" indentLevel="3" synch="#t02622878"
+                who="#WU00018921">
+                <p> Wie du siehst hab ich die Lemma geändert, danke für den Hinweis, ich war nämlich
+                    selbst auch etwas unsicher bei der ganzen Sache und bin jetzt damit auch viel
+                    glücklicher!--</p><signed rend="inline" generatedBy="template"><gap
+                        reason="signatureContent"/>
+                    <time generatedBy="template">12:01, 12. Jun. 2009 (CEST)</time>
+                </signed>
+            </post>
+        </egXML>
+    </exemplum>
+    <exemplum type="Blog" xml:lang="en" versionDate="2021-09-07">
+        <p>Blog comment: Die Anthropodizee-Frage. Wer den Himmel leerräumt, schafft die Menschheit
+            ab. by Volker Birk.
+                <ref>https://scilogs.spektrum.de/natur-des-glaubens/die-anthropodizee-frage-wer-himmel/#comment-31899</ref></p>
+
+        <egXML xmlns="http://www.tei-c.org/ns/Examples">
+            <post xml:id="p5" type="comment" who="#u4" synch="#t005" replyTo="#p4">
+                <p>“Wenn Sie diesen Gruppen also “mangelnde Bildung“ attestieren wollen, so
+                    verwenden Sie bereits einen bestimmten, kulturgebundenen Bildungsbegriff.”</p>
+                <p>Ich hoffe doch, wir können beim Bildungsbegriff der Aufklärung bleiben. Wer das
+                    nicht möchte, hat die Wissenschaft verlassen.</p></post>
+        </egXML>
+    </exemplum>
+    <exemplum type="usenet" xml:lang="en" versionDate="2024-07-02">
+        <p>Usenet reply that comments on material quoted in an earlier message.</p>
+        <egXML xmlns="http://www.tei-c.org/ns/Examples" source="#usenet-sgml-tei" >
+            <post when="1990-10-11" who="#dmerritt" xml:id="sgml-tei-3" replyTo="#sgml-tei-1">
+                <time generatedBy="system">04:37:51 UTC</time>
+                <p>In article <ref target="#sgml-tei-1">&lt;2****.**********0@m*******.***.*****.**c.uk&gt;</ref> S.*.*.*****z@e**.*****.**c.uk (Sebastian Rahtz) writes:
```

Comment:
I would add a `<seg>` to wrap: In article <ref target="#sgml-tei-1">&lt;2****.**********0@m*******.***.*****.**c.uk&gt;</ref> S.*.*.*****z@e**.*****.**c.uk (Sebastian Rahtz) writes:` with the `@generatedBy="system"` attribute.